App.js

```diff
@@ -1,30 +1,42 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
-import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client';
-import { settings } from './config'
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
+import { settings } from "./config";
 
 import Login from "./Pages/Login";
 import Register from "./Pages/Register";
-import ResetPassword from "./Pages/ResetPassword"
+import ResetPassword from "./Pages/ResetPassword";
+import CodeButton from "./Pages/CodeButton";
 
 const Stack = createStackNavigator();
 
 const client = new ApolloClient({
-  uri: `http://${settings.internalIP}:5000`,  
+  uri: `http://${settings.internalIP}:5000`,
   cache: new InMemoryCache(),
 });
 
 export default function App() {
+  //hard-coded user that will be passed by points page
+  let user = {
```

Comment:
can you remove stuff like this from the App.js, since we will plug in your button code into App.js later